### DISCUSSION TITLE
Heads-up: applying clang-format to the repository

### DISCUSSION THREAD

**Discussion by kxl-adsk** [Autodesk/maya-usd#873] [Announcement]
We are going to apply clang-format to [files](https://github.com/Autodesk/maya-usd/blob/dev/.clang-format-include) in the maya-usd repository. This is scheduled to happen the week of November 2nd.

We mention it here since it has at least two important implications for some of you:
* contributors will have to apply clang-format (version `10.0.0`) for future pull-requests
* anyone having outstanding branches will need to follow instructions from #843

Let us know if you have any concerns or questions.

**Comment by rlei-weta** [comment]
That's a great addition!!

**Comment by kxl-adsk** [comment]
Clang-format has been applied to `dev` - see #890 for instructions on how to merge `dev` into outstanding branches.
